akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala

```diff
@@ -104,12 +104,18 @@ class FileAndResourceDirectivesSpec extends RoutingSpec with Inspectors with Ins
     "reject requests to directory resources with trailing slash" in {
       Get() ~> getFromResource("someDir/") ~> check { handled shouldEqual (false) }
     }
-    "reject requests to directory resources from an Archive " in {
+    "reject requests to directory resources from an archive " in {
       Get() ~> getFromResource("com/typesafe/config") ~> check { handled shouldEqual (false) }
     }
-    "reject requests to directory resources from an Archive with trailing slash" in {
+    "reject requests to directory resources from an archive with trailing slash" in {
       Get() ~> getFromResource("com/typesafe/config/") ~> check { handled shouldEqual (false) }
     }
+    "return the resource from an archive with spaces and umlauts" in {
+      Get() ~> getFromResource("test-resource.txt") ~> check {
```

Comment:
Maybe add a comment that it will load this file from the jar file above. Easy enough to spot in the diff but not so easy if you just look at the test.